recipe/meta.yaml

```diff
@@ -24,15 +24,19 @@ requirements:
     - cross-r-base {{ r_base }}  # [build_platform != target_platform]
     - r-nanotime  # [build_platform != target_platform]
     - r-rcpp >=1.0.8  # [build_platform != target_platform]
+    - r-rcppspdlog  # [build_platform != target_platform]
```

Comment:
This was the missing line that broke the cross-compiled builds on osx-arm64 and linux-aarch64

Reply:
That would do it for not being able to load a wrong binary :)    In Debian-speak 'binary-all' vs 'binary-any', or in R-speak 'native code' vs 'no native code'.  Nicely done.  Glad this is solved!